tendrl/commons/jobs/__init__.py

```diff
@@ -56,18 +60,32 @@ def _process_job(self, raw_job, job_key):
     def _acceptor(self):
         while not self.job_consumer_thread._complete.is_set():
             gevent.sleep(2)
-
             # TODO(team) replace below raw write with a "EtcdJobQueue" class
             try:
                 jobs = tendrl_ns.etcd_orm.client.read("/queue")
             except etcd.EtcdKeyNotFound:
                 continue
-                
             for job in jobs.children:
-                executed = False
-                if job.value is None:
+                try:
+                    raw_job = {"job_id": None,
+                               "integration_id": None,
+                               "run": None,
+                               "status": None,
+                               "parameters": None,
+                               "type": None,
+                               "node_ids": None,
+                               "request_id": None,
+                               "errors": None,
+                               "parent": None}
+                    job_id = job.key.split("/")[-1]
```

Comment:
@r0h4n api is creating job_id in url like queue/<job_id> so i need job id the process further, 
so when is read /queue it gives all job_keys like /queue/<job_id> . so i am split job_id from the key.